neo4j-ruby-driver.gemspec

```diff
@@ -33,32 +33,34 @@ Gem::Specification.new do |spec|
     `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   end
 
-  ffi = ENV['SEABOLT_LIB']&.length&.positive?
-  pdir = ffi ? 'ffi' : 'jruby'
+  mri = !RUBY_PLATFORM.match?(/java/)
+  pdir = mri ? 'ffi' : ENV.has_key?('SEABOLT_LIB') ? 'ffi' : 'jruby'
 
   spec.files << Dir['lib/**/*.rb']
   spec.files << Dir["#{pdir}/**/*.rb"]
-  spec.files << Dir["#{pdir}/**/*.jar"]
+
+  unless mri
+    spec.files << Dir["#{pdir}/**/*.jar"]
+    spec.platform = 'java'
+  end
```

Comment:
the whole above refactoring is incorrect. You are including the jar files even in case of ffi on jruby